src/core/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -146,7 +146,19 @@
 	                IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.DisableDriveDetectionWarning}"
 	                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher}"
 	                />
+            <TextBlock x:Uid="PowerLauncher_Theme"
+                       Style="{StaticResource SettingsGroupTitleStyle}"

```

Comment:
This should be a settings group-section title. We do did for e.g. shortcut guide "Appearance".

Change the UID into:

x:Uid="ShortcutGuide_Appearance_Behavior"

Reply:
@niels9001
I think you mean `x:Uid="PowerLau cher_Appearance_Behavior"`

Reply:
Corrected, it should be: Appearance_GroupSettings.

It already exists in the Resources file for other pages.